tests/phpunit/tests/rest-api/rest-themes-controller.php

```diff
@@ -471,6 +475,24 @@ public function test_theme_requires_wp() {
 		$this->assertSame( '5.3', $result[0]['requires_wp'] );
 	}
 
+	public function test_theme_is_block_theme() {
+		switch_theme( 'block-theme' );
+
+		$response = self::perform_active_theme_request();
+		$result   = $response->get_data();
+
+		$this->assertTrue( isset( $result[0]['is_block_theme'] ) );
+		$this->assertTrue( $result[0]['is_block_theme'] );
+	}
+
+	public function test_theme_is_not_block_theme() {
+		$response = self::perform_active_theme_request();
+		$result   = $response->get_data();
+
+		$this->assertTrue( isset( $result[0]['is_block_theme'] ) );
```

Comment:
`assertTrue()` may be a little obscure here. Other tests use `assertArrayHasKey()`, for example.

This may also need to be preceded by `assertIsArray( $result )` and `assertNotEmpty( $result )` to prevent errors in the test suite should `$response->get_data()` ever return a non-array, or an empty array.

Reply:
Thanks for the feedback @costdev !

I updated to use `assertArrayHasKey` in https://github.com/WordPress/wordpress-develop/pull/4323/commits/9c5c62279d8f19fe07a08815963f5d4b43ce31f0.

> This may also need to be preceded by assertIsArray( $result ) and assertNotEmpty( $result ) to prevent errors in the test suite should $response->get_data() ever return a non-array, or an empty array.

Good point! Given that $data is assumed to contain a result in several tests in this file, I added these assertions to the more general tests `test_get_items` and `test_get_items_inactive` in https://github.com/WordPress/wordpress-develop/pull/4323/commits/f2e7ac75bb9c31318df5243c4b437ecb1f18727e. Any further refactoring of the tests here seems out of scope for this PR.